dakara_player_vlc/dakara_server.py

```diff
@@ -220,21 +218,28 @@ def send_status_get_commands(self, playing_id, timing=0, paused=False):
             return response.json()
 
         logger.error("Unable to send status to server")
-        logger.debug("""Error code: {code}
-Message: {message}""".format(
+        logger.debug("Error {code}: {message}".format(
             code=response.status_code,
-            message=response.text
-            ))
+            message=display_message(response.text)
+        ))
 
         return {'pause': True, 'skip': False}
 
 
+def display_message(message, limit=100):
+    """Display the 100 first characters of a message
+    """
+    if len(message) <= limit:
+        return message
+
+    return message[:limit].strip() + "..."
+
+
```

Comment:
So, now the log messages are limited in length
what is the motivation behind this change?

Reply:
My bad, I forgot to list this feature.

Sometimes in development when there are errors when communicating with the server, the response of the server is an entire HTML page, which floods the logs.

Reply:
Maybe I should shrink in case of HTML responde only ?